components/centraldashboard/public/components/main-page.js

```diff
@@ -29,7 +29,7 @@ import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
 import css from './main-page.css';
 import template from './main-page.pug';
 import logo from '../assets/logo.svg';
-import '../assets/anon-user.png';
+import '../assets/logout.svg';
```

Comment:
Don't need it, since the icon bundler already imports it

```suggestion
import css from './main-page.css';
import template from './main-page.pug';
import logo from '../assets/logo.svg';
```